chart/values.yaml

```diff
@@ -349,6 +349,8 @@ csi:
       ctrl_loss_tmo: "1980"
       # Kato (keep alive timeout) in seconds
       keep_alive_tmo: ""
+      # Fallback to nvme connect over tcp if host is not rdma capable
+      nvme_connect_fallback: false
```

Comment:
This should be true by default?

Reply:
> This should be true by default?

The default is kept `false` to let users choose explicitly if they wish to fallback connect, and not taking a decision on their behalf.

Reply:
As discussed, made it `true` for now.